.prettierignore

```diff
@@ -3,3 +3,4 @@
 .reports
 coverage
 node_modules
+docs
```

Comment:
Why are we ignoring docs for prettier? This contains instructions, etc.. that can be prettyfied too, for example we have the api spec, etc.. and I believe pettier will format yml files etc...

Reply:
Do we need to configure prettier for yml files? It was formatting api-spec and changing the quotation marks.

Reply:
This is the default config since we didn't provide overrides: https://prettier.io/docs/en/options.html
however overrides are possible to if we wish to have double, single quote for whatever file, it can be done.

Reply:
What is the agreed way forward for this? @JonLazariniDD

Reply:
I think it needs to be prettified but whether we use single or double quote for string, I am not sure whats DVSA preference but it doesn't matter to me as they are both strings and will depend what character you want to espace when not using template strings (``). 
If we wanted to tweak the config for yml files we can follow this example and adapt it: https://prettier.io/docs/en/configuration.html#configuration-overrides